src/main/scala/actors/PlutoUpdaterActor.scala

```diff
@@ -0,0 +1,69 @@
+package actors
+
+import java.time.LocalDateTime
+import java.util.Base64
+
+import actors.messages._
+import akka.actor.{Actor, ActorRef, Props}
+import akka.event.{DiagnosticLoggingAdapter, Logging}
+import akka.pattern.ask
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{Keep, Sink, Source}
+import akka.util.ByteString
+import com.gu.contentatom.thrift.{Atom, AtomData}
+import com.softwaremill.sttp.akkahttp.AkkaHttpBackend
+import com.softwaremill.sttp._
+import com.typesafe.config.Config
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+import vidispine.{UpdateXmlGenerator, VSCommunicator, VSError, VSSearchResponse}
+
+class PlutoUpdaterActor(config:Config) extends Actor with VSCommunicator{
```

Comment:
What's the reason for using actors here? You don't seem to be maintaining any state, so you might get cleaner code by just using futures

Reply:
yeah, I tried it that way, but seemed to be simpler to use Akka HTTP from within an actor, at least that is the way that all of the examples went.  So I gave it a whirl and was able to bring the app together much quicker that way.